pkgs/development/python-modules/pylev/default.nix

```diff
@@ -0,0 +1,18 @@
+{ lib, buildPythonPackage, fetchPypi }:
+buildPythonPackage rec {
+  pname = "pylev";
+  version = "1.3.0";
+
+  src = fetchPypi {
```

Comment:
I'm not seeing the test in pypi tarball. I don't think we should go through any trouble to get it though.

So the `checkPhase` should be disabled with

```nix
doCheck = false;
```